### DISCUSSION TITLE
Parsing a Path parameter with a trailing extension

### DISCUSSION THREAD

**Discussion by phaer** [poem-web/poem#380] [Q&A]
Hello,

I am trying to use a Route like `"/notebooks/:id.pdf` but, it seems that poetry interprets this as a parameter called "id.pdf" instead of an parameter "id", followed by the string ".pdf". Using a regex like `/notebooks/:id<[^\\.]>.pdf` does not work as expected either. 

Swagger UI confirms that the parameters aren't interpreted as expected: 
![image](https://user-images.githubusercontent.com/101753/187967937-f80a65f5-9676-44a6-9e49-cd62fd1f8046.png)

Is this something supported by poem? Am I missing something?

**Comment by sunli829** [comment]
https://github.com/poem-web/poem/issues/381
